backend/src/libs/helpers/get-sorted-by.helper.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  type AnyColumn,
+  type SQL,
+  type SQLWrapper,
+  asc,
+  desc,
+} from 'drizzle-orm';
+
+import { SortMethod } from '../enums/enums.js';
+import { type SortMethodValue } from '../types/types.js';
+
+type ColumnToSorted = SQLWrapper | AnyColumn;
+
+const getSortedBy = (
+  sortedMethod: SortMethodValue,
+  requiredColumnToSorted: ColumnToSorted,
+  optionalColumnToSorted: ColumnToSorted,
+): SQL[] => {
```

Comment:
can't we ORDER BY in query ?

Reply:
We get two different types of sorting from the frontend, and in this helper, I determine which type I have, and we create a specific query to the database based on that. I think it can be useful not only for me, and there will be a little less code in the repositories.